### ISSUE TITLE
training target

### ISSUE THREAD

**Issue by 1450081092** [tsinghua-fib-lab/LCSim#9]
Hey there, 

I trained the model for 200 epochs with `module_vel` as the training target, same as the project's approach. But the generated trajectories show issues like the vehicle wobbling in place and high collision rates—way less realistic than the examples in the code. 

`    def _get_training_targets(self, data: HeteroData) -> torch.Tensor:
        if self.target == "post_vel":
            traj = data["agent"]["target"][..., : self.output_dim]
            # diff
            gt = torch.cat((traj[:, :1], traj[:, 1:] - traj[:, :-1]), dim=1)
        elif self.target == "pca":
            traj = data["agent"]["target"][..., : self.input_dim]
            traj = traj.reshape(-1, self.num_future_steps * self.input_dim)
            gt = self.pca(traj)
        elif self.target == "module_vel":
            traj = data["agent"]["target"][..., : self.input_dim]
            vel = torch.cat(
                (traj[:, :1], traj[:, 1:] - traj[:, :-1]),
                dim=1,
            )
            module = torch.norm(vel, p=2, dim=-1, keepdim=True) - VEL_MEAN
            angle = torch.atan2(vel[..., 1], vel[..., 0]).unsqueeze(-1)
            gt = torch.cat((module, angle), dim=-1)
        else:
            raise ValueError(f"Unknown target type: {self.target}")
        return gt`

Just wondering which training target was used to get the results shown in the official examples? Any tips on training configs or hyperparameters would be super helpful!
Thanks!
